### PR TITLE
updated main script to argparse, started minor testing

### PR DESCRIPTION
**Pull request by jennyfolkesson**

### CODE REVIEW

File: tests/run_array_analyzer_tests.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import os
+from testfixtures import TempDirectory
```
Comment (bryantChhun) [inline tests/run_array_analyzer_tests.py:2]: @jennyfolkesson Should we add testfixtures as a dependency?

**Comment by mattersoflight** [comment]
I just tested the CLI, it works well.

**Comment by mattersoflight** [comment]
Good idea to setup tests. I've started a feature request for integration test (#39). @jennyfolkesson it'd make sense to add test folder to master once implemented. But, feel free to merge as is too.

**Comment by bryantChhun** [comment]
I've also set up cli and it seems to work.  There's extra configuring that needs to be done to run this from pycharm IDE.

Also, I set up the following shell script that will allow us to run as before.  It can be left out of the repo, but is possibly useful for others:

```
#!/bin/sh

#runs three tests under test folder

# Fluplate - old imaged on scienion
# INPUT_PATH=/Volumes/GoogleDrive/My\ Drive/ELISAarrayReader/images_scienion/Plates_given_to_manu/2020-01-15_plate4_AEP_Feb3_6mousesera

# Fluplate - old imaged on octopi
# INPUT_PATH=/Volumes/GoogleDrive/My\ Drive/ELISAarrayReader/images_octopi/20200325AdamsPlate/Averaged/500us

# Fluplate - new imaged on scienion 3/30/2020
# INPUT_PATH=/Volumes/GoogleDrive/My\ Drive/ELISAarrayReader/images_scienion/2020-03-30-15-08-05-COVD_March25_fluplatetest_AdamsPlate

# Fluplate - blinded test imaged on scienion 4/4/2020
INPUT_PATH=/Volumes/GoogleDrive/My\ Drive/ELISAarrayReader/images_scienion/2020-04-04-14-18-32-COVID_April4_flusecondplate

# Write results to local folder, copy them to drive if useful.
DATA_DIR=$(python -c "import os; print(os.path.basename(os.path.normpath('$INPUT_PATH')))")
OUTPUT_PATH=$(python -c "import os; print(os.path.join(os.path.expanduser('~/Documents/images_local/'),'$DATA_DIR'))")

python run_array_analyzer.py -i "$INPUT_PATH" -o "$OUTPUT_PATH" -m 'fit' -d
```

This script is placed at the same level as "run_array_analyzer.py" and executed with just 
```"sh <this_script.sh>"```

**Comment by bryantChhun** [comment]
Other than the question above about test dependency, I think this is good to merge.

**Review by bryantChhun — APPROVED** [review]

**Pull request merged**